src/core/control/XournalMain.cpp

```diff
@@ -408,15 +408,30 @@ void on_startup(GApplication* application, XMPtr app_data) {
 
     // Set up icons
     {
-        const auto lightIcons = app_data->gladePath->getFirstSearchPath() / "iconsColor-light";
-        const auto darkIcons = app_data->gladePath->getFirstSearchPath() / "iconsColor-dark";
+        const auto uiPath = app_data->gladePath->getFirstSearchPath();
+        const auto lightIcons = (uiPath / "iconsColor-light").u8string();
+        const auto darkIcons = (uiPath / "iconsColor-dark").u8string();
+        const auto lightLucideIcons = (uiPath / "iconsLucide-light").u8string();
+        const auto darkLucideIcons = (uiPath / "iconsLucide-dark").u8string();
 
         // icon load order from lowest priority to highest priority
-        std::vector<std::string> iconLoadOrder = {lightIcons.u8string()};
-        if (app_data->control->getSettings()->isDarkTheme()) {
-            iconLoadOrder.emplace_back(darkIcons.u8string());
-        } else {
-            iconLoadOrder.insert(iconLoadOrder.begin(), darkIcons.u8string());
+        std::vector<std::string> iconLoadOrder = {};
```

Comment:
Can you add a comment clarifying that _later_ items in the icon load order have _higher_ priority?

Reply:
There is already a comment, just the line above.

Reply:
Somehow I missed that, thanks.